app/resources/views/pages/dominion/op-center/index.blade.php

```diff
@@ -93,6 +93,61 @@
                     </table>
                 </div>
             </div>
+
+            <div class="box box-primary">
+                <div class="box-header">
+                    <h3 class="box-title">Town Crier</h3>
```

Comment:
Maybe call it something else?

Reply:
Agree, I'm not really happy with that title.

Reply:
Maybe just 'Clairvoyance'?